docs/src/modules/components/ExamplesGrid.js

```diff
@@ -0,0 +1,134 @@
+      source: 'https://github.com/mui/mui-toolpad/tree/master/examples/basic-crud-app',
+    },
+    {
+      title: 'QR Code generator',
+      description:
+        'A basic Toolpad application that can be used to turn any text or URL into a QR code.',
+      src: '/static/toolpad/docs/examples/qr-generator.png',
+      href: 'https://mui.com/toolpad/examples/qr-generator/',
+      source: 'https://github.com/mui/mui-toolpad/tree/master/examples/qr-generator',
+    },
+    {
+      title: 'With Prisma',
+      description: 'A basic Toolpad application that demonstrates how to integrate with prisma.',
+      src: '/static/toolpad/marketing/with-prisma-hero.png',
+      source: 'https://github.com/mui/mui-toolpad/tree/master/examples/with-prisma',
+    },
+    {
+      title: 'Google sheet',
+      description: 'Quickly fetch data from Google sheets to build a Toolpad app',
+      src: '/static/toolpad/marketing/google-sheet.png',
+      href: 'https://mui.com/toolpad/how-to-guides/connect-to-googlesheets/',
+      source: 'https://github.com/mui/mui-toolpad/tree/master/examples/google-sheet',
+    },
+    {
+      title: 'Dog app',
+      description: 'An app that shows dog images based on selected breeds or sub-breeds.',
+      src: '/static/toolpad/docs/getting-started/first-app/step-13.png',
+      source: 'https://github.com/mui/mui-toolpad/tree/master/examples/dog-app',
+    },
+    {
+      title: 'Data grid columns',
+      description: 'A basic Toolpad app that shows how to add a code component inside a Data Grid',
+      src: '/static/toolpad/marketing/datagrid-columns.png',
+      source: 'https://github.com/mui/mui-toolpad/tree/master/examples/datagrid-columns',
+    },
+    {
+      title: 'Custom data grid',
+      description: 'A basic Toolpad app that shows how to add a code component inside a Data Grid',
+      src: '/static/toolpad/marketing/custom-datagrid-column.png',
+      source: 'https://github.com/mui/mui-toolpad/tree/master/examples/custom-datagrid-column',
+    },
+  ];
+}
+
+function Templates() {
+  return (
+    <Grid container spacing={2} sx={{ pt: 2, pb: 4 }}>
+      {layouts().map((layout) => (
+        <Grid item xs={12} sm={4} sx={{ flexGrow: 1 }} key={layout.title}>
+          <Card
+            sx={(theme) => ({
+              height: '100%',
+              display: 'flex',
+              flexDirection: 'column',
+              px: 2,
+              pt: 2,
+              pb: 1,
+              gap: 1.5,
+              borderRadius: 1,
+              backgroundColor: `${alpha(theme.palette.grey[50], 0.4)}`,
+              borderColor: 'divider',
+              ...theme.applyDarkStyles({
+                backgroundColor: `${alpha(theme.palette.primaryDark[700], 0.3)}`,
+                borderColor: 'divider',
+              }),
+            })}
+            variant="outlined"
+          >
+            <CardMedia
+              component="a"
+              image={layout.src}
+              title={layout.title}
```

Comment:
Probably want this to be 

```suggestion
              title={layout.description}
```
as this ends up being the tooltip.